src/components/Sidebar.tsx

```diff
@@ -1,4 +1,4 @@
-import type { FC } from 'react';
+import { useEffect, type FC } from 'react';
```

Comment:
This is not necessary but for your own interest.

I tend to keep the type imports separate. As I find it easier to read. See:

```suggestion
import { useEffect } from 'react';
...

import type { FC } from 'react';
```